pages/work/[id].tsx

```diff
@@ -97,9 +101,26 @@ const WorkDetailPage = ({ id, modeStr, workPublic }: WorkDetailPageProps) => {
           作品一覧に戻る
         </ButtonLink>
         {workDetail.authors.map((a) => a.userId).includes(authState.user.userId) && (
-          <Button variant="contained" color="error" onClick={onClickDelete} startIcon={<Delete />}>
-            削除する
-          </Button>
+          <Stack direction="row" spacing={2}>
+            {modeStr === "edit" ? null : (
+              <Button
+                variant="contained"
+                color="primary"
+                onClick={onClickEdit}
+                startIcon={<Edit />}
+              >
+                編集する
+              </Button>
+            )}
```

Comment:
三項演算子で片方が `null` になる場合は論理積演算子が使えるので、こちらを使うようにしてほしいです。

```suggestion
            {modeStr !== "edit" && (
              <Button
                variant="contained"
                color="primary"
                onClick={onClickEdit}
                startIcon={<Edit />}
              >
                編集する
              </Button>
            )}
```